### PR TITLE
Add ProTip to remind users to grant the app access to the org

### PR DESCRIPTION
**Pull request by tarebyte**
/cc @johndbritton for copy

### CODE REVIEW

**Comment by tarebyte** [comment]
screen shot 2015-09-22 at 13 46 59

File: app/views/organizations/new.html.erb [hunk 1]
```diff
@@ -32,5 +32,9 @@
         </div>
       <% end %>
     <% end %>
+
+    <div class="protip">
+      <%= octicon('light-bulb') %> <strong>ProTip!</strong> Don't see your organization? <a href="https://github.com/settings/applications">Make sure you have granted us access.</a>
```
Comment (johndbritton) [inline app/views/organizations/new.html.erb]: > Don't see the organization you're looking for? Make sure you've [granted](https://github.com/settings/applications) the Classroom for GitHub application access to your organization.

Comment (johndbritton) [inline app/views/organizations/new.html.erb]: Better yet, you can link to the exact page using this: https://developer.github.com/v3/oauth/#directing-users-to-review-their-access-for-an-application

**Comment by tarebyte** [comment]
screen shot 2015-09-22 at 15 52 23

**Comment by johndbritton** [comment]
Can we move the ProTip out of the `.site-content-body` div?

File: app/views/organizations/new.html.erb [hunk 2]
```diff
@@ -32,5 +32,9 @@
         </div>
       <% end %>
     <% end %>
+
+    <div class="protip">
+      <%= octicon('light-bulb') %> <strong>ProTip!</strong> Don't see the organization you're looking for? Make sure you've <%= link_to 'granted', "https://github.com/settings/connections/applications/#{Rails.application.secrets.github_client_id}" %> the Classroom for GitHub application access to your organization.
```
Comment (johndbritton) [inline app/views/organizations/new.html.erb]: This is too long, sorry for being verbose.

> Don't see your organization? You need to [grant access access](#) first.

**Comment by tarebyte** [comment]
screen shot 2015-09-22 at 16 01 12

**Pull request merged**